changelog.md

```diff
@@ -1,5 +1,6 @@
 # Changelog
 - 1.2.2
+   - [PR #240 - Fix build error, update compiler checks](https://github.com/ProjectDent/ARKit-CoreLocation/pull/240)
```

Comment:
Per previous guidance from @intere, we don't need Changelog entries for changes that don't modify library features.